client/hosting/sites/components/dotcom-style.scss

```diff
@@ -102,27 +102,16 @@
 
 	}
 
-	.dataviews-filters__view-actions {
```

Comment:
I wonder if we can also remove these `.dataviews-filters__view-actions` selectors in `components/style.css` as well?